cmd/agent/app/integrations.go

```diff
@@ -27,7 +27,6 @@ import (
 )
 
 const (
-	tufConfigFile       = "public-tuf-config.json"
 	reqAgentReleaseFile = "requirements-agent-release.txt"
 	constraintsFile     = "final_constraints.txt"
 	tufPkgPattern       = "datadog(-|_).*"
```

Comment:
```suggestion
	tufPkgPattern       = "datadog-.*"
```

Reply:
Because we are not using `pip` to download anymore, we will not get "autocorrection."